tensorwaves/optimizer/minuit.py

```diff
@@ -0,0 +1,71 @@
+"""Minuit2 adapter to the iminuit package."""
```

Comment:
Intersphinx reference to `iminuit`'s API?
https://iminuit.readthedocs.io/en/latest/

Reply:
True